hack/code/prices_gen.go

```diff
@@ -93,7 +93,7 @@ func main() {
 	os.Setenv("AWS_SDK_LOAD_CONFIG", "true")
 	os.Setenv("AWS_REGION", region)
 	ctx := context.Background()
-	ctx = settings.ToContext(ctx, test.Settings())
+	ctx = options.ToContext(ctx, test.Options())
```

Comment:
Ah, I see you why you renamed this the first time now. This isn't blocking and I'm fine either way that you choose this, but I'd personally prefer that we keep the struct and function `Options` and `NewOptions`

Reply:
This didn't actually end up being the issue, I just left it after I changed it since it seemed more descriptive anyway. I'm perfectly happy to change it back.

Reply:
So a problem is multiple files in `hack/code` have an `Options` struct and they're all in the main package so we have name conflicts. I'm not sure what a good solution here is other than:
a) Separate out hack files into separate folders
b) Add qualifiers the names (e.g. PriceGenOptions, VPCLimitsOptions)